.github/workflows/e2e-starter.yml

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: E2E Starter
+
+on:
+  # run against every merge commit to 'main' and release branches
+  push:
+    branches:
+      - main
+      - release-*
+  # only run on PRs that touch certain regex paths
+  pull_request:
+    branches:
+      - main
+      - release-*
+    paths:
+      # note this should match the merging criteria in 'mergify.yml'
+      - '**.py'
+      - 'pyproject.toml'
+      - 'requirements**.txt'
+      - 'scripts/basic-workflow-tests.sh' # Used by this workflow
+      - 'scripts/test-data/**'
+      - '.github/workflows/e2e-nvidia-t4-x1-gate.yml' # This workflow
```

Comment:
```suggestion
      - '.github/workflows/e2e-starter.yml' # This workflow
```
old name?

Reply:
I specifically left the name of the actual workflow so that it changes, the starter workflow will run. If the starter workflow changes there is no real need to run the e2e.

Reply:
What I'm saying is that there is no file named `e2e-nvidia-t4-x1-gate`: https://github.com/bjhargrave/instructlab_instructlab/tree/issues/2245/.github/workflows

So, maybe it's supposed to be `e2e-nvidia-t4-x1.yml`?

Reply:
No worries @bjhargrave we can wait til Wed, enjoy your time off!